python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -338,9 +338,14 @@ def makePlot(
                         extent=extent,
                         rcparams=rcparams,
                     )
-                    sp.draw_hspmap(mapData, zoom=zoom)
-                    sp.set_xlabel("RA")
-                    sp.set_ylabel("Dec")
+                    # Work around skyproj bug that will fail to zoom on empty
+                    # map.
+                    if mapData.n_valid == 0:
+                        sp.draw_hspmap(mapData, zoom=False)
+                    else:
+                        sp.draw_hspmap(mapData, zoom=zoom)
+                    sp.ax.set_xlabel("RA")
+                    sp.ax.set_ylabel("Dec")
```

Comment:
Ah, I will drop [this commit](https://github.com/lsst/analysis_tools/pull/314/commits/fca0ec210533efe44ec0ea1f773c4ed98625c429) from my PR (#314) then.